src/Loggers/OwnPatternFormatter.cpp

```diff
@@ -63,6 +65,8 @@ void OwnPatternFormatter::formatExtended(const DB::ExtendedLogMessage & msg_ext,
         writeCString(resetColor(), wb);
     writeCString(": ", wb);
     DB::writeString(msg.getText(), wb);
+
+    wb.finalize();
```

Comment:
I wonder why this code is using so much `WriteBufferFromString` and so little fmt::format

Reply:
We just append strings or chars so not much point in `fmt::format`